examples/add_instance.rb

```diff
@@ -18,8 +19,10 @@
   
 end.parse!
 
-# Token comes for `~/.eyrc`
-client = Ey::Core::Client.new(token: "abcdefghijklmnrstuvwxyz123456789")
+# Token comes from '~/.eyrc'
+eyrc = YAML.load_file(File.expand_path("~/.eyrc"))
+
+client = Ey::Core::Client.new(token: eyrc['api_token'])
```

Comment:
i dont like the behavior of overriding the `.ey-core` file.  especially as that file is going to replace `.eyrc` soon, but I will go with it for now.